samples/error-correction/syndrome/Syndrome.csproj

```
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.Quantum.Sdk/0.12.20070124">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
```

Since you have no operation in your `.qs` file marked as `@EntryPoint()`, this should probably be a library project rather than an executable project.
```suggestion
  <PropertyGroup>
    <TargetFramework>netstandard2.1</TargetFramework>
  </PropertyGroup>
```
I'm assuming you only added this file so that you get IntelliSense, etc.? As a complete aside, once https://github.com/microsoft/qsharp-compiler/pull/560 is done, you'll be able to get language server functionality in VS/VS Code without creating a `.csproj` at all.

---

Great point. I added this because I was initially planning to have an `Entrypoint`, but then decided against it. Indeed, I could just leave it out. Thanks for the suggestion.